spec/models/audit_log_spec.rb

```diff
@@ -2,5 +2,4 @@
 
 RSpec.describe AuditLog, type: :model do
   it { is_expected.to belong_to(:user) }
```

Comment:
This test seems to failing in CI? :thinking:

Reply:
great catch! fixed it! I also brought in the latest master to remove the parallel build